clouddriver-core/src/test/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionAuthorizerSpec.groovy

```diff
@@ -68,4 +100,22 @@ class DescriptionAuthorizerSpec extends Specification {
       this.names = names
     }
   }
+
+  class TestImageTaggingDescription implements AccountNameable {
```

Comment:
I couldn't access the actual implementations because `clouddriver-core` doesn't depend on the aws or google modules. Do you prefer this test class implementation, or should I depend on the aws and google modules using gradle `testImplementation`?

Reply:
I prefer this rather than adding the dependency, thanks!